### ISSUE TITLE
Error: Argument lengths do not match

### ISSUE THREAD

**Issue by Ricc4rdo0107** [klattimer/LGWebOSRemote#119]
`lgtv TV notification hi`
`Error: Argument lengths do not match`

**Comment by egnappahz** [comment]
I agree with this, had to update/rebuild this app for pything 3.11 and something has changed and broken. I dont know about previous changes here but the only thing I know is that this fixes the problem:

in "lib/python3.11/site-packages/LGT/__init__.py" (or "master/LGTV/__init__.py")  on line 168:
`
    elif len(sys.argv) >= 2 and sys.argv[2] == "on":
        name = sys.argv[1]
        ws = LGTVRemote(name, **config[name])
        ws.on()
        sleep(1)
        sys.exit(0)
    else:
        try:
            **#args = parseargs(sys.argv[2], sys.argv[4:])**
            **args = parseargs(sys.argv[2], sys.argv[3:])**
            name = sys.argv[1]
            command = sys.argv[2]
        except Exception as e:
            usage(str(e))
            sys.exit(1)
`

For some reason it needs to be the third argument instead of the 4th. It was just a dirty fix from my part, i don't know half of the flow of this program, just wanted to get it fixed asap for myself without losing too much time, but something is definetly not right here.

**Comment by gdegas** [comment]
Or you can just write the argument twice if it's easier `lgtv mytv mute true true`
